### PR TITLE
feat(Search): noResultsMessage prop can be a node

### PR DESCRIPTION
**Pull request by b0gok**
PR for this issue https://github.com/Semantic-Org/Semantic-UI-React/issues/1664

### CODE REVIEW

File: src/modules/Search/Search.js [hunk 1]
```diff
@@ -57,7 +57,10 @@ export default class Search extends Component {
     noResultsDescription: PropTypes.string,
 
     /** Message to display when there are no results. */
-    noResultsMessage: PropTypes.string,
+    noResultsMessage: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.object,
+    ]),
```
Comment (levithomason) [inline src/modules/Search/Search.js]: I believe this should just be `PropTypes.node` only.  `PropTypes.node` includes any renderable value: string, number, element, or an array (or fragment) of any of these.  However, objects are not renderable.

Let's also update the typings in `Search.d.ts` to reflect the type change as well.

**Review by levithomason — CHANGES_REQUESTED** [review]
Just one comment regarding propTypes and typings.

File: src/modules/Search/Search.js (hunk 1 above)
Comment (b0gok) [reply src/modules/Search/Search.js]: > PropTypes.node includes any renderable value: string, number, element, or an array (or fragment) of any of these. 

Didn't know about it. Thanks for useful information.
Made the necessary changes.

File: src/modules/Search/Search.d.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export interface SearchProps {
   noResultsDescription?: string;
```
Comment (layershifter) [inline src/modules/Search/Search.d.ts]: @b0gok Thanks for a PR. I think that we need make same changes for `noResultsDescription`.

**Review by layershifter — CHANGES_REQUESTED** [review]

File: src/modules/Search/Search.d.ts (hunk 2 above)
Comment (b0gok) [reply src/modules/Search/Search.d.ts]: @layershifter Should I add specific test for case when noResultsDescription isn't a string?

Comment (layershifter) [reply src/modules/Search/Search.d.ts]: Yes, it will be awesome.

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Thank you for the PR and updates!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.4`.

**Pull request merged**